Assets/Scripts/AI/RacerManager.cs

```diff
@@ -29,34 +36,39 @@ private void Start()
     private void Update()
     {
         UpdatePositions();
+        if (!raceFinished)
+        {
+            timeElapsed = Time.time - startTime;
```

Comment:
The idea of the start time variable was to avoid having more variables and only compute the time elapsed when needed using Time.time

If you want to update it like that you should remove startTime as a variable and do "timeElapsed += Time.deltaTime" to update the timeElapsed

Reply:
Note that this is a minor issue and isn't a big deal